description/audiovisual.shacl.ttl

```diff
@@ -1,23 +1,22 @@
 @prefix dct: <http://purl.org/dc/terms/> .
-@prefix haDes: <http://data.hetarchief.be/ns/description/> .
+@prefix haDes: <https://data.hetarchief.be/ns/description/> .
 @prefix pav: <http://purl.org/pav/> .
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 @prefix vann: <http://purl.org/vocab/vann/> .
-
 @prefix ebucore: <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
 @prefix haObj: <https://data.hetarchief.be/ns/object/> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix schema: <https://schema.org/> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
-
-
 @prefix rel: <http://id.loc.gov/vocabulary/preservation/relationshipSubType/> .
 @prefix premis: <http://www.loc.gov/premis/rdf/v3/> .
 
-<http://data.hetarchief.be/ns/description/>  a owl:Ontology ;
+@base <https://data.hetarchief.be/ns/audiovisual> . 
+
+<>  a owl:Ontology ;
     vann:preferredNamespacePrefix "haDes" ;
```

Comment:
Should we also change this to a separate prefix then? The specific ontology terms are part of the general ontology 'haDes', but the data model is separate. I don't know what makes the most sense anymore :D

Reply:
see comment in https://meemoo.atlassian.net/browse/KG-262?focusedCommentId=46431 Not sure either 😄

Reply:
I went for a compromise and removed them. They were currently not serving any purpose and they can be brought back if they do.